vendor/imports.go

```diff
@@ -50,20 +50,33 @@ func ParseImports(root string) (map[string]bool, error) {
 
 // FetchMetadata fetchs the remote metadata for path.
 func FetchMetadata(path string, insecure bool) (io.ReadCloser, error) {
-	schemes := []string{"https", "http"}
+	var schemes []string
```

Comment:
please try this instead

var schemes = []string { "https" }
if insecure {
     schemes = append(schemes, "http")
}